client/components/webpack-build-monitor/index.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-console */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { createStore } from 'redux';
+import cx from 'classnames';
```

Comment:
i can change that, for some reason my instinct i always need to fight is to name it `cx`.